src/modules/FFmpeg/FFDecSW.cpp

```diff
@@ -339,8 +339,6 @@ bool FFDecSW::open(StreamInfo &streamInfo, VideoWriter *)
 		return false;
 	if (codec_ctx->codec_type == AVMEDIA_TYPE_VIDEO)
 	{
-		if (codec->capabilities & CODEC_CAP_DR1)
```

Comment:
look at this [link](https://github.com/FFmpeg/FFmpeg/blob/n2.2/libavcodec/avcodec.h#L738)